src/common/logging/default-logger.ts

```diff
@@ -4,7 +4,11 @@ import { Logger } from './logger';
 // Licensed under the MIT License.
 export const createDefaultLogger = (): Logger => {
     return {
-        log: (message?: any, ...otherArgs: any[]) => console.log(message, ...otherArgs),
-        error: (message?: any, ...otherArgs: any[]) => console.error(message, ...otherArgs),
+        // It's important that these use the console functions directly rather than wrapping them
```

Comment:
base on the comments on the test itself, I don't think you need this comments... looks like over documenting why you're doing this.
Also, how about typings for `Logger`? does the types show up properly on VSCode?

Reply:
Fair enough, removing redundant comment. The types still show up fine in VSCode at the consuming classes (they're all written in terms of the `Logger` type, which this PR doesn't impact)